#
# This Solaris specific patch provides for possibility to build
# OpenSSH with openssl-3 libraries located in non-standard place,
# specified via configure option --with-ssl-dir.
#
--- openssh-9.4p1/configure.ac
+++ openssh-9.4p1/configure.ac
@@ -2730,8 +2730,8 @@ case "$withval" in
 				# Relative paths
 				./*|../*)	withval="`pwd`/$withval"
 			esac
-			if test -d "$withval/lib"; then
-				libcrypto_path="${withval}/lib"
+			if test -d "$withval/lib/64"; then
+				libcrypto_path="${withval}/lib/64"
 			elif test -d "$withval/lib64"; then
 				libcrypto_path="$withval/lib64"
 			else
